db/seeds.rb

```diff
@@ -218,7 +218,7 @@
 10.times do
   pet = Pet.create!(
     organization: Organization.all.sample,
-    name: Faker::Creature::Dog.name,
+    name: Faker::Creature::Dog.unique.name,
```

Comment:
We may not want to use `.unique` here. It slows it down a bit and we really want to remove the uniqueness